lib/listen/silencer.rb

```diff
@@ -57,23 +57,21 @@ class Silencer
       | ~
     )$}x.freeze
 
+    # TODO: deprecate these mutators; use attr_reader instead
     attr_accessor :only_patterns, :ignore_patterns
 
-    def initialize
-      configure({})
+    def initialize(options = {})
```

Comment:
I'm not sure the backwards compatibility constraints but you should prefer `**options`.

Reply:
All previous cases initialized this object with no arguments, so I don't think there's a concern there.

Can you help me understand what cases will work better with `**options` vs. `options = {}`?

Reply:
I know that `**options` is effective at collecting the options hash as the last argument when there are many arguments. But with a single argument like this, I'm not seeing a difference between `**options` and `options = {}`?

Reply:
Maybe `**options` is preferred because it always works regardless of other arguments before it?
I switched over to it here: https://github.com/guard/listen/commit/45921334d45e7c09bfca4c29c6e3abd8ba79e632